docs/documentation/indexing/fast_fields.mdx

```diff
@@ -5,25 +5,28 @@ title: Fast Fields
 ## Basic Usage
 
 A field that is indexed as `fast` is stored in a column-oriented fashion. Fast fields are necessary for
-[aggregations/faceting](/documentation/faceting/overview). They can also improve the query times of [filtering](/documentation/full-text/filtering)
-and BM25 scoring.
+[aggregations/faceting](/documentation/faceting/overview). They can also improve the query times of [filtering](/documentation/full-text/filtering) and
+[sorting](/documentation/full-text/sorting).
 
-The following code block demonstrates how to specify a fast field.
+By default, [numeric](/documentation/indexing/create_index#numeric-fields), [datetime](/documentation/indexing/create_index#datetime-fields), and [boolean](/documentation/indexing/create_index#boolean-fields)
+are indexed as fast. The following code block demonstrates how to specify a fast field.
 
 ```sql
 CALL paradedb.create_bm25(
   index_name => 'search_idx',
   table_name => 'mock_items',
   key_field => 'id',
-  numeric_fields => paradedb.field('rating', fast => true)
+  numeric_fields => paradedb.field('rating', fast => true | false)
```

Comment:
Is this `| false` intentional?

Reply:
That came from @rebasedming.  Probably not tho.  I'll fix.